src/sql/workbench/parts/tasks/browser/tasksRenderer.ts

```diff
@@ -108,15 +112,15 @@ export class TaskHistoryRenderer implements IRenderer {
 			templateData.description.textContent = description;
 			templateData.description.title = templateData.description.textContent;
 
-			this.timer(element, templateData);
-			let self = this;
-			setInterval(function () {
-				self.timer(element, templateData);
-			}, 1000);
+			this.timer(element, templateData.time);
+
+			const timer = new IntervalTimer();
+			timer.cancelAndSet(() => this.timer(element, templateData.time), 500);
```

Comment:
it looks like the timer is now `500` instead of `1000`.  Any reason for this (I would expect increasing update frequency to increase flicker)?

Reply:
The change was arbitrary. I just rewrote this section and set it to w/e looked good to me but I can revert to 1000